### PR TITLE
Firstbranch

### PR DESCRIPTION
**Pull request by randallwvr90**
Hey!

This is a pull request which means I made a branch and now I think we should merge that branch with the main branch. The main branch is where the "real" code lives and new branches are an easy way to try new code. Then once you are happy with the new code, you merge your branch with the main branch. I added you as a reviewer so you can look and we can discuss the change before merging with the main branch. 

Anyway, I just made two files: musicplayer and drumplayer. They only have comments so far with a high-level description of what they would do. Do you think this is a good way to do things?

### CODE REVIEW

File: ux/musicplayer.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*
+*************************************************************************************************************************
+**** This Java class will handle playing the drums. ****
+
+	What kind of methods does it need?
+	- play_drums
+		- it could have some kind of global variable that will increase or decrease the volume and/or speed. 
+		  Maybe one for each? volumen and speed?
+		- Will it play drums for a certain amount of time? Will it play drums until it gets some other input?
```
Comment (JimWeaverOK) [inline ux/musicplayer.java]: I image some patterns or cadences, too

File: ux/musicplayer.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/*
+*************************************************************************************************************************
+**** This Java class will handle playing the drums. ****
+
+	What kind of methods does it need?
+	- play_drums
+		- it could have some kind of global variable that will increase or decrease the volume and/or speed. 
+		  Maybe one for each? volumen and speed?
```
Comment (JimWeaverOK) [inline ux/musicplayer.java]: volume and speed could be separate for max flexibility

File: ux/musicplayer.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+		- Will it play drums for a certain amount of time? Will it play drums until it gets some other input?
+	- change_volume
+	- change_speed
+*************************************************************************************************************************
+*/
```
Comment (JimWeaverOK) [inline ux/musicplayer.java:12]: I'm thinking of having the drums either respond to a green house gas calculation and also play at random intervals.  Intensity is determined by the greeenhouse calculator, or is determined randomly.

Comment (randallwvr90) [reply ux/musicplayer.java:12]: I was thinking there will be some other object that "decides" what it should tell the output objects to do based on the greenhouse calculator. Or I guess the output objects could keep checking to see when the calculation changes and then change their behavior accordingly.

**Pull request closed without merging**